datasets/integration_tests/pytorch/pytorch_test.py

```diff
@@ -0,0 +1,131 @@
+import unittest
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torch.optim as optim
+from datasets.utils.logging import disable_progress_bar
+from parameterized import parameterized_class, parameterized
+from torch import Tensor
+from torch.utils.data import DataLoader
+from torchvision.transforms import Compose, ToTensor, Normalize
+
+from flwr_datasets import FederatedDataset
+
+
+class SimpleCNN(nn.Module):
+    def __init__(self):
+        super(SimpleCNN, self).__init__()
+        self.conv1 = nn.Conv2d(3, 6, 5)
+        self.pool = nn.MaxPool2d(2, 2)
+        self.conv2 = nn.Conv2d(6, 16, 5)
+        self.fc1 = nn.Linear(16 * 5 * 5, 120)
+        self.fc2 = nn.Linear(120, 84)
+        self.fc3 = nn.Linear(84, 10)
+
+    def forward(self, x):
+        x = self.pool(F.relu(self.conv1(x)))
+        x = self.pool(F.relu(self.conv2(x)))
+        x = x.view(-1, 16 * 5 * 5)
+        x = F.relu(self.fc1(x))
+        x = F.relu(self.fc2(x))
+        x = self.fc3(x)
+        return x
+
+
+# Using parameterized testing, two different sets of parameters are specified:
+# 1. CIFAR10 dataset with the simple ToTensor transform.
+# 2. CIFAR10 dataset with a composed transform that first converts an image to a tensor
+#    and then normalizes it.
+@parameterized_class(
+    [
+        {"dataset_name": "cifar10", "test_split": "test", "transforms": ToTensor()},
+        {"dataset_name": "cifar10", "test_split": "test", "transforms": Compose(
+            [ToTensor(), Normalize((0.5, 0.5, 0.5), (0.5, 0.5, 0.5))]
+        )},
+    ]
+)
+class FDSToPyTorchCorrectUsage(unittest.TestCase):
```

Comment:
We usually CamelCase abbreviations:
```suggestion
class FdsToPyTorchCorrectUsage(unittest.TestCase):
```